### PR TITLE
Readme md analytics

### PR DESCRIPTION
**Pull request by juanmaguitar**
**What**:

Adding README.md's of analytic packages and templates to create similar README.md's for the rest of the packages

**Why**:

So every package has the proper info in NPM

**How**:

Creating the README.md's using on each folder

```
npx readme-md-generator -p ../../.github/README_TEMPLATE/package-readme.md -y
```

And then adding custom info on each README.md

**Tasks**:

**Unrelated Tasks**

- Code
- TSDocs
- TypeScript
- Unit tests
- End to end tests
- TypeScript tests
- Update starter themes
- Update other packages
- Link to PR in [Documentation](https://github.com/frontity/gitbook-docs/)
- Community discussions
- Changeset (with link to its [Feature Discussion](https://community.frontity.org/c/33) if it exists)

**Additional Comments**

### CODE REVIEW

File: packages/google-analytics/README.md [hunk 1]
```diff
@@ -1,10 +1,67 @@
...
+
+**`frontity.settings.js`**
+```js
+export default {
+  packages: [
+    {
+      name: "@frontity/google-analytics",
+      state: {
+        googleAnalytics: {
+          trackingIds: ['UA-12345678-9']
```
Comment (luisherranz) [inline packages/google-analytics/README.md]: For the basic usage I would show the singular option `trackingId`:
- https://github.com/frontity/frontity/blob/dev/packages/google-analytics/types.ts#L30

The plural `trackingIds` is only needed for advanced sites that need more than one tracking ID.

File: packages/google-tag-manager/README.md [hunk 2]
```diff
@@ -1,13 +1,67 @@
...
+
+**`frontity.settings.js`**
+```js
+export default {
+  packages: [
+    {
+      name: "@frontity/google-tag-manager",
+      state: {
+        googleTagManager: {
+          containerIds: ['GTM-BCDFGHJ']
```
Comment (luisherranz) [inline packages/google-tag-manager/README.md]: For the basic usage I would show the singular option `containerId`:

- https://github.com/frontity/frontity/blob/dev/packages/google-tag-manager/types.ts#L19

The plural `containerIds` is only needed for advanced sites that need more than one container ID.

File: packages/google-tag-manager/README.md [hunk 3]
```diff
@@ -1,13 +1,67 @@
-> ## Beta version
->
-> Please, bear in mind that this package is currently in beta version. It hasn't proper tests or docs and its implementation may change with the final release.
```
Comment (luisherranz) [inline packages/google-tag-manager/README.md]: We still need to maintain a warning about the beta version.

File: packages/google-analytics/README.md [hunk 4]
```diff
@@ -1,10 +1,67 @@
-> ## Beta version
-> Please, bear in mind that this package is currently in beta version. It hasn't proper tests or docs and its implementation may change with the final release.
```
Comment (luisherranz) [inline packages/google-analytics/README.md]: We still need to maintain a warning about the beta version.

**Review by luisherranz — CHANGES_REQUESTED** [review]
They are great, thanks @juanma 👏👏

BTW, what is your idea for the Usage section? Add a basic Usage section here and refer to the docs for complete information?

**Comment by SantosGuillamot** [comment]
I really like them, thanks @juanmaguitar ! :slightly_smiling_face: I'm just missing some links to the docs, but I guess it's because we still don't have that pages created.

Btw, quick question, if we change something in the footer template we still need to update the packages manually right?

**Comment by juanmaguitar** [comment]
> BTW, what is your idea for the Usage section? Add a basic Usage section here and refer to the docs for complete information?

Yes! Adding some basic example of how to use the package and then adding a link to the docs to more advanced usage

**Comment by juanmaguitar** [comment]
> I really like them, thanks @juanmaguitar ! 🙂 I'm just missing some links to the docs, but I guess it's because we still don't have that pages created.

Yes, these packages doesn't have specific documentation in the docs. Those whoe have it will have the proper link to the docs

> Btw, quick question, if we change something in the footer template we still need to update the packages manually right?

You're right. The templates are used to create new `README.md`. 
Any update on a `README.md` has to be done manually or re-creating a new README.md using the templates and then updating it

File: packages/google-tag-manager/README.md (hunk 2 above)
Comment (juanmaguitar) [reply packages/google-tag-manager/README.md]: Done! 👍

File: packages/google-tag-manager/README.md (hunk 3 above)
Comment (juanmaguitar) [reply packages/google-tag-manager/README.md]: Done! 👍

File: packages/google-analytics/README.md (hunk 4 above)
Comment (juanmaguitar) [reply packages/google-analytics/README.md]: Done! 👍

File: packages/google-analytics/README.md (hunk 1 above)
Comment (juanmaguitar) [reply packages/google-analytics/README.md]: Done! 👍

**Review by luisherranz — APPROVED** [review]
Thanks @juanmaguitar  👍

**Review by mburridge — APPROVED** [review]

**Comment by juanmaguitar** [comment]
@luisherranz Is this ready to be merged?

**Comment by luisherranz** [comment]
Sure. It's approved 😄

_(I've rerun the tests because an e2e test failed. We'll have to check that.)_

**Pull request merged**